target/linux/rtl838x/files-5.4/drivers/net/ethernet/rtl838x_eth.c

```diff
@@ -0,0 +1,1205 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
Get `ERROR: do not use C99 // comments` here as well. However, I don't know what's the proper way for the SPDX license in c files.